test-case/check-sof-logger.sh

```diff
@@ -161,6 +159,14 @@ run_loggers()
              > "$data_file" 2> "$error_file" & dmaPID=$!
     fi
 
+    sleep "$logger_settle_time_secs"
+    # Avoid racing with runtime-pm by opening the PCM nodes when
+    # the loggers are running.
+    ping_pcm_devices
+
+    dlogi "Collecting logs now, last kernel activity:"
+    journalctl_cmd |grep "kernel: sof" |tail -1 || true
```

Comment:
```suggestion
    journalctl_cmd -g ' sof' | tail -n 3
```
There is already a `-k` in `journalctl_cmd()`.

`tail` always succeeds (unless some true disaster happened, and then we want to stop)

Reply:
Updated this as well. I had "kernel: sof" on purpose to not matching random " sof" strings in kernel output, but after a few tests, this seems good enough.